### PR TITLE
DOC-4187 Add cluster-health.md

### PR DESCRIPTION
**Pull request by davidcurrie**
Add content to `cluster-health.md` (this section could equally be called "Troubleshooting Deployment"). Also added a section to the support bundles topic on setting log levels.

### CODE REVIEW

**Comment by davidcurrie** [comment]
@AnnaK-R3 - as promised!

**Comment by davidcurrie** [comment]
@AnnaK-R3 / @nadinequinr3  - I'm just going to assume that you have an alternative route for getting these changes in...

**Comment by nadinequinr3** [comment]
@davidcurrie can you try merging this into the [DOC-4318-Corda5Alpa](https://github.com/corda/corda-docs-portal/tree/DOC-4318-Corda5Alpa) branch?

**Comment by davidcurrie** [comment]
@nadinequinr3 - rebased on `DOC-4318-Corda5Alpa`

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/cluster-health.md [hunk 1]
```diff
@@ -9,3 +9,93 @@ menu:
...
+
+If a pod has restarted, it may be worth looking at the logs from the terminated instance using the `--previous` option.
+For example:
+
+```shell
+kubectl logs --previous corda-db-worker-f9994f756-tqtx7
+```
+
+The type of the failing work may also give a clue to the root cause.
+For example, all of the workers connect to Kafka but only the DB and Crypto workers connect to PostgreSQL.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-alpha/deploying/troubleshooting/cluster-health.md:81]:
```suggestion
For example, all of the workers connect to Kafka but only the database and Crypto workers connect to PostgreSQL.
```

File: content/en/platform/corda/5.0-alpha/deploying/troubleshooting/cluster-health.md [hunk 2]
```diff
@@ -9,3 +9,93 @@ menu:
...
+Failures in the pods for these two workers may therefore indicate an issue with PostgreSQL.
+
+## Post-install Job
+
+Once all of the workers are ready, the Helm install will execute a post-install job.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-alpha/deploying/troubleshooting/cluster-health.md:86]:
```suggestion
Once all of the workers are ready, the Helm install executes a post-install job.
```

**Review by AnnaK-R3 — COMMENTED** [review]
@davidcurrie left some editorial comments. Please review and commit suggestions and I'll merge this PR.

**Comment by AnnaK-R3** [comment]
@davidcurrie I managed to commit the changes myself. Merged your content. Thanks for your contribution to the docs!

**Review by yarivus — COMMENTED** [review]
thanks @AnnaK-R3

**Pull request merged**